### PR TITLE
Fixes issue #696

### PR DESCRIPTION
**Pull request by pliu2781**
I edited the code so that, if l_abstract passes a --loc_inst to a presentation that has both 'location' and 'institution', then location will take priority. If a presentation only has 'institution', then the program will only lists presentations that are seminars/ colliquims

### CODE REVIEW

**Comment by sbillinge** [comment]
I think we want it to throw an exception if it finds both.  We don't want to introduce magic.

btw, something is still wrong with travis...it is not triggering builds.

**Comment by pliu2781** [comment]
I edited the code so that it displays a warning for presentations that have loc_inst in both location and institution. Then it prints those presentations.

**Comment by pliu2781** [comment]
Is the use of warnings okay? Or should I have it throw an exception instead?

**Comment by sbillinge** [comment]
exceptio

> 
> 
> Is the use of warnings okay? Or should I have it throw an exception instead?

exception.  Then it won't pass CI and the person will have to fix the problem.

File: regolith/helpers/l_abstracthelper.py [hunk 1]
```diff
@@ -103,18 +102,15 @@ def sout(self):
...
+                            if rc.loc_inst.casefold() in presentation.get('location',"").casefold()]
             if exceptions:
-                warn("These presentations have %s in both 'location' and 'institution'" % (rc.loc_inst))
-            for presentation in exceptions:
-                print("---------------------------------------")
-                print(f"Title: {presentation.get('title')}")
-                print(f"location: {presentation.get('location')}")
-                print(f"institution: {presentation.get('institution')}")
+                for presentation in exceptions:
+                    raise Exception("This presentation has %s in both 'location' and 'institution'" % (rc.loc_inst),
```
Comment (sbillinge) [inline regolith/helpers/l_abstracthelper.py:110]: - [ ] use an `fstring`
- [ ] raise ValueError
- [ ] tell the user the offending presentation, not just "This presentation" if you can
- [ ] move it up tp the top so it takes less time to crash if it is going to crash
- [ ] it is ok for this to be here (this is a lister), but it is more useful making sure the adder/updaters don't allow it in the first place.  They may not, but now is a great time to check while we are thinking about it.
- [ ] it needs a test

One more thought.  If we are even more clever we can do this check in the database validation.  google for cerberus and "conditional validation" or sthg like that.  Then we don't need any of htis code, which is kind of ugly anyway....

That is much more elegant a solution so worth looking into then we can just remove all this.

**Review by sbillinge — COMMENTED** [review]
some thougthts, but at the end an idea that might be better....

File: regolith/helpers/l_abstracthelper.py (hunk 1 above)
Comment (pliu2781) [reply regolith/helpers/l_abstracthelper.py:110]: Should I implement the cerberus solution in l_abstracthelper.py or in schemas.py?

Comment (sbillinge) [reply regolith/helpers/l_abstracthelper.py:110]: schemas.py.  It may also be necessary to tweak `regolith.validate`

**Comment by pliu2781** [comment]
Closing this PR, as the cerberus fix makes this redundant

**Pull request merged**